test/e2e/specs/all-features/recordset/saved-query.spec.ts

```diff
@@ -0,0 +1,254 @@
+import { test, expect } from '@playwright/test';
+import moment from 'moment';
+
+// locators
+import AlertLocators from '@isrd-isi-edu/chaise/test/e2e/locators/alert';
+import ModalLocators from '@isrd-isi-edu/chaise/test/e2e/locators/modal';
+import RecordLocators from '@isrd-isi-edu/chaise/test/e2e/locators/record';
+import RecordeditLocators from '@isrd-isi-edu/chaise/test/e2e/locators/recordedit';
+import RecordsetLocators from '@isrd-isi-edu/chaise/test/e2e/locators/recordset';
+
+// utils
+import { getCatalogID, getEntityRow, updateCatalogAnnotation } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
+import { testShareCiteModal } from '@isrd-isi-edu/chaise/test/e2e/utils/record-utils';
+import { clearInput } from '@isrd-isi-edu/chaise/test/e2e/utils/recordedit-utils';
+
+const testParams = {
+  table_name: 'main',
+  minInputClass: 'range-min',
+  minInputClearClass: 'min-clear',
+  maxInputClass: 'range-max',
+  maxInputClearClass: 'max-clear',
+  firstSavedQueryName: 'main with int_col ( 11 to 22)',
+  secondSavedQueryName: 'main with int_col ( 11 to 22); one',
+  key: {
+    name: 'id',
+    operator: '=',
+    value: '1'
+  }
+};
+
+test.describe('View recordset page and form a query,', () => {
```

Comment:
This is something that I always assumed that we need, but we don't have to have one outer `test.describe` on our tests. We can have multiple or none at all.

In this case given that the share cite test is opening a completely different page, I think it's better if you move it outside the `describe` like I think you originally planned to do. 

```ts
test.describe('saved query feature', () => {
  ...
})

test('citation on the share and cite modal', ...)
```